src/notification.js

```diff
@@ -89,67 +90,69 @@ class Notification extends Component {
       letterSpacing: '.125ex',
       textTransform: 'uppercase',
       borderRadius: '5px',
-      cursor: 'pointer'
+      cursor: 'pointer',
     }, this.props.actionStyle) : {};
   }
 
   /*
-   * @function getTitleStyle
-   * @description Dynamically get the styles for the title.
-   * @returns {object} result The style.
-   */
+  * @function getTitleStyle
+  * @description Dynamically get the styles for the title.
+  * @returns {object} result The style.
+  */
   getTitleStyle() {
     return this.props.style !== false ? Object.assign({}, {
       fontWeight: '700',
-      marginRight: '.5rem'
+      marginRight: '.5rem',
     }, this.props.titleStyle) : {};
   }
 
   /*
-   * @function handleClick
-   * @description Handle click events on the action button.
-   */
+  * @function handleClick
+  * @description Handle click events on the action button.
+  */
   handleClick() {
     if (this.props.onClick && typeof this.props.onClick === 'function') {
       return this.props.onClick();
     }
+
+    return () => {};
   }
 
   render() {
     let className = 'notification-bar';
 
-    if (this.props.isActive) className += ' ' + this.props.activeClassName;
-    if (this.props.className) className += ' ' + this.props.className;
+    if (this.props.isActive) className += ` ${this.props.activeClassName}`;
+    if (this.props.className) className += ` ${this.props.className}`;
 
     return (
       <div className={className} style={this.getBarStyle()}>
-        <div className="notification-bar-wrapper">
-          {this.props.title ? (
-            <span
-              ref="title"
-              className="notification-bar-title"
-              style={this.getTitleStyle()}
-            >
-              {this.props.title}
-            </span>
-          ) : null}
-          <span
-            ref="message"
-            className="notification-bar-message"
-          >
-            {this.props.message}
-          </span>
-          {this.props.action ? (
-            <span
-              ref="action"
-              className="notification-bar-action"
-              onClick={this.handleClick}
-              style={this.getActionStyle()}
-            >
-              {this.props.action}
-            </span>
-          ) : null}
-        </div>
+      <div className="notification-bar-wrapper">
```

Comment:
This `div` is a child of the `div` above and should be indent once to show that.